Cargo.toml

```diff
@@ -27,7 +28,7 @@ builder = ["dep:typed-builder"]
 [dependencies]
 typed-builder = { version = "0.14.0", optional = true }
 serde         = { version = "1.0.159", features = ["derive"] }
-reqwest       = { version = "0.11.16", features = ["json"] }
+reqwest       = { version = "0.11.16", default-features = false, optional = true }
```

Comment:
Don't make reqwest optional. This will break compilation. Just disable  default-features and add charset, http2, macos-system-configuration and json to reqwest features.

Reply:
We might have to update the version to add `charset` feature. Details [here](https://github.com/seanmonstar/reqwest/blob/master/CHANGELOG.md#v0120).